### PR TITLE
[issue#41_Scripts_tensorboard] [NF]:Tensorbaord charts

### LINKED ISSUE #41: Добавить вывод графиков к скрипту обучения
Чтобы более наглядно следить за тем, что происходит с моделью в процессе обучения - удобно использовать графики. 

`from torch.utils.tensorboard import SummaryWriter` - https://pytorch.org/docs/stable/tensorboard.html
Нужно в твой скрипт обучения встроить эту штуку, в результате ты должна получить следующие графики: 
- функции потерь при train 
- функции потерь при valid 
- Изменение learning rate (здесь есть маленькая хитрость, нужно погуглить :smiley_cat: ) 

> Модуль tensorboard будет рисовать графики тебе в браузер в реальном времени, при желании их можно будет сохранить себе локально. Но главное, что это наглядно, а трудозатрат минимально (=

### PR DESCRIPTION
**Pull request by MarinaKupetskova**

### CODE REVIEW

File: Scripts/mnist_train.py [hunk 1]
```diff
@@ -10,11 +10,14 @@
 import torch.nn as nn
 import torchvision
 from torch.optim.lr_scheduler import ReduceLROnPlateau
+from torch.utils.tensorboard import SummaryWriter
 from torchvision.transforms import transforms
 from tqdm import tqdm
 
 from object_detection.mnist_model import MNIST
 
+writer = SummaryWriter()
```
Comment (serykhelena) [inline Scripts/mnist_train.py]: это можно внутрь функции main, глобально ведь его не используем

**Review by serykhelena — APPROVED** [review]
